schemas/validation_tests/ArrayAssay-invalid-2.json

```diff
@@ -0,0 +1,66 @@
+{
+    "object_ids": {
+        "alias" : "MyArrayAssay", 
+        "center_name" : "EBI-TEST",
+        "accession": "EGAA00000000001"
+    },
+    "arrayAssay_center": "EBI-TEST",
+    "arrayAssay_date": "2021-06-24",
+    "sample_number": 2,
+    "sample_Labels": ["EGAN00000000001:Cy3", "EGAN00000000002:Biotin"],
+    "ArrayAssay_relationships": [
+        {
+            "r_type": "Referenced_in",
+            "r_source": "EGAA00002045411",
+            "r_sourceType": "ArrayAssay",
+            "r_target": "EGAE00000000001", 
+            "r_targetType": "ArrayExperiment"
+        },
+        {
+            "r_type": "Same_as [NCIT:C64637]",
+            "r_source": "EGAA00000000001",
+            "r_sourceType": "ArrayAssay",
+            "r_target": "EGAA00000000002", 
+            "r_targetType": "ArrayAssay"
+        },
+        {
+            "r_source": "EGAN00000000001",
+            "r_sourceType": "sample",
+            "r_type": "Referenced_in",
+            "r_target": "EGAA00000000001", 
+            "r_targetType": "ArrayAssay"
+        }
+    ],
+    "assay_files": [
+        {
+            "filename": "myfile1.cel",
+            "filetype": "CEL [EFO:0005630]",
```

Comment:
Here I have the answer of one my comments above. Assay files will have both, filename and filetype. Are we going to use a controlled vocabulary for file types, or are we going to accept any filetype (as it is now). Just curious :)

Reply:
If you take a look at ``EGA.common-definitions.json`` > ``definitions`` > ``file_object`` > ``filetype`` you will see that, at least for now, the approach is to only allow certain filetypes (those within the ``enum`` clause):

```` json
            "enum": ["CEL [EFO:0005630]", "TSV [NCIT:C164049]", "ADF [NCIT:C172213]", "FASTQ [EFO:0004155]", "FASTA [NCIT:C47845]", "SDRF [NCIT:C172211]", "IDF [NCIT:C172212]", "VCF [NCIT:C172216]", "SRA [format:3698]", "SRF [EFO:0004154]", "SFF [EFO:0004156]", "BAM [EFO:0004157]", "CRAM [format:3462]", "XLSX [format:3620]", "CSV [format:3752]"]
````